SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/EventTracingTests/UserProcessCrossServerPropertyEventTracingTest.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "UserProcessCrossServerPropertyEventTracingTest.h"
+
+AUserProcessCrossServerPropertyEventTracingTest::AUserProcessCrossServerPropertyEventTracingTest()
+{
+	Author = "Danny Birch";
+	Description = TEXT("Test checking user event traces can be caused by receive property update events for cross-server properties");
+
+	FilterEventNames = { UserReceiveCrossServerPropertyEventName, ReceivePropertyUpdateEventName };
+	WorkerDefinition = FWorkerDefinition::Client(1);
+}
+
+void AUserProcessCrossServerPropertyEventTracingTest::FinishEventTraceTest()
+{
+	int EventsTested = 0;
+	int EventsFailed = 0;
+	for (const auto& Pair : TraceEvents)
+	{
+		const FString& SpanIdString = Pair.Key;
+		const FName& EventName = Pair.Value;
+
+		if (EventName != UserReceiveCrossServerPropertyEventName)
```

Comment:
Does this not want to be ReceivePropertyUpdateEventName?

Reply:
hmm this is the receive > user so we want to test UserReceiveCrossServerPropertyEventName causes below?